receiver/splunkhecreceiver/receiver.go

```diff
@@ -443,3 +450,11 @@ func initJSONResponse(s string) []byte {
 	}
 	return respBody
 }
+
+func appendMetadataToResource(resource pcommon.Resource, req *http.Request, param ...string) {
+	for _, k := range param {
+		if q := req.URL.Query().Get(k); q != "" {
+			resource.Attributes().InsertString(k, q)
```

Comment:
this is a bit too open ended. Can we limit to the ones we explicitly want to support?

Also, we map those to attributes defined here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/splunk/common.go#L30

Reply:
changed to use the default attribute keys